frontend/components/styles/Nav.tsx

```diff
@@ -72,37 +41,62 @@ const NavItem = ({
   title: string
   link: string
 }) => {
+  const router = useRouter()
+  const isActive = router.pathname === link
+
   return (
     <Link href={link}>
-      <NavItemWrapper>
-        <Text heading>
-          <Icon name={icon} margin="auto 1rem 1px 0" />
+      <div
+        className={`flex gap-2 cursor-pointer items-center py-2 px-4 rounded-full border ${
+          isActive
+            ? 'bg-neutral-50 shadow-sm shadow-neutral-100'
+            : 'border-transparent'
+        }`}
+      >
+        <img
+          src={iconSrcMap[icon] || '/icons/dashboard.svg'}
+          alt={`${icon} icon`}
+          width={18}
+          height={18}
+          className={isActive ? 'text-blue-500' : 'text-gray-500'}
+        />
+        <span
+          className={`text-base font-work-sans ${
+            isActive ? 'text-blue-500' : 'text-neutral-500'
+          }`}
+        >
           {title}
-        </Text>
-      </NavItemWrapper>
+        </span>
+      </div>
     </Link>
   )
 }
 
-const NavWrapper = s.div`
-  width: ${NAV_WIDTH};
-  background-color: ${colors.NAV_BACKGROUND};
-  text-align: center;
-  min-height: 99vh;
-  position: fixed;
-`
-
 export const Nav = () => {
   const { user } = useContext(AuthUserContext)
 
   return (
-    <NavWrapper>
+    <div
+      className={`w-[12rem] bg-${colors.NAV_BACKGROUND} flex flex-col justify-between items-center min-h-screen fixed py-4`}
```

Comment:
same thing about color here? 

also why not move `fixed` over and `text-align`
also why was before min-height 99vh? 
answer could just be you have no clue but the new thing works

Reply:
I think the 99vh was supposed to give a little bit of wiggle room for spacing, and it didn't matter because everything was bg-white.

Eventually, I should add in a plugin that will sort Tailwind classes automatically upon save.